src/fu-engine.c

```diff
@@ -3786,6 +3786,8 @@ fu_common_device_category_to_name(const gchar *cat)
 		return "Mouse";
 	if (g_strcmp0(cat, "X-Keyboard") == 0)
 		return "Keyboard";
+	if (g_strcmp0(cat, "X-VideoDisplay") == 0)
+		return "Video Display";
```

Comment:
Maybe just Display? at least to my ears `Video Display` seems a bit odd to hear in 2022.

Reply:
It was to match the icon name.

okay for the string.

Reply:
done.